frontend/src/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Typography, Box, Container } from '@mui/material';
+
+import Button from 'components/Button';
+
+const ListItem: React.FC = () => (
+  <Container
+    style={{
+      borderRadius: '15px',
+      border: '1px solid tomato',
+      // width: '500px',
+      // maxHeight: '600px',
+      display: 'flex',
+      flexDirection: 'column',
+      padding: '5px',
+    }}
+  >
+    <Box style={{ width: '100%' }}>
+      <Typography
+        variant="body2"
+        style={{ margin: '0px', textAlign: 'justify', width: '100%', marginBottom: '20px' }}
```

Comment:
Remove styles please
Also why do you use <img> inside of Typography? There can only be text in typography
Please use MUI Box this way:
`Box component="img" src={image} alt="your-img-alt"`
Import image from images folder please

Reply:
Deleted as useless